manageiq-schema.gemspec

```
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency "ancestry"
   spec.add_dependency "activerecord-id_regions", "~> 0.3.2"
-  spec.add_dependency "linux_admin",             "~> 2.0"
+  spec.add_dependency "linux_admin",             "~> 3.0"
```

Should we loosen this dependency?
```suggestion
  spec.add_dependency "linux_admin",             ">= 2.0", "< 4"
```

---

Done